source/common/quic/envoy_quic_server_connection.cc

```diff
@@ -50,6 +52,17 @@ void EnvoyQuicServerConnection::OnCanWrite() {
   onWriteEventDone();
 }
 
+void EnvoyQuicServerConnection::ProcessUdpPacket(const quic::QuicSocketAddress& self_address,
+                                                 const quic::QuicSocketAddress& peer_address,
+                                                 const quic::QuicReceivedPacket& packet) {
+  if (!first_packet_received_) {
+    listener_filter_manager_->onFirstPacketReceived(packet);
+  }
+  first_packet_received_ = true;
```

Comment:
nits: move this into the if block above?

Reply:
done!